### ISSUE TITLE
Add to Live USB / Use with fresh install ?

### ISSUE THREAD

**Issue by jayesbe** [thopiekar/rcraid-dkms#12]
Hi, thank you for this repo.  However, I am having an issue with a brand new install and just wondering if there is something I'm doing incorrectly or if I am missing something.

I am trying to setup a fresh raid0 (Asus x470 Crosshair VII).  I have two nvme drives (samsung evo pro) and have setup the array in the bios.

I can boot into the Ubuntu Live USB key I created.. however then comes the blocking issue.  GParted does not recognize the raid array.. 

I tried then adding in this repo and installing mdadm.  Unfortunately that too didn't get me anywhere.. 

Thanks for any assistance anyone can provide.

**Comment by thopiekar** [comment]
https://thopiekar.eu/other/amd-raidxpress-for-ubuntu-debian-packaging-and-patches/

Regarding step 9: You can either rmmod as described or blacklist the ahci module before booting the live OS.

**Comment by jayesbe** [comment]
I tried that, however I came to the conclusion the issue is the new UEFI 9.2 driver. 

I have a X470 motherboard which I was forced to bios update in order to even boot the machine for the new 3800X processor.  That bios update installed UEFI 9.2 firmware.  Which only supports Windows 10 at the moment.  The rcraid driver currently being applied via this package is 8.1 I think? .. in any case it's incompatible with the bios update.  A new driver is required.
